### PR TITLE
feat: setup authentication system phase 2

### PR DESCRIPTION
**Pull request by seanho96**
This PR generates the devise bootstrap views and adds in the necessary connections to the manifest.

### CODE REVIEW

**Review by tlkiong — COMMENTED** [review]
PR is super big. Make it hard to review effectively without putting significant effort.

**Comment by seanho96** [comment]
@tlkiong My bad, will split it up 🙏

File: app/views/devise/shared/_links.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
+  <%- if controller_name != 'sessions' %>
+    <%= link_to t(".sign_in"), new_session_path(resource_name) %><br />
+  <% end -%>
```
Comment (tlkiong) [inline app/views/devise/shared/_links.html.erb]: Why this file has a mix of `-` at the beginning & `-` end?

**Review by tlkiong — APPROVED** [review]
small Q but LGTM 🚀

File: app/views/devise/shared/_links.html.erb (hunk 1 above)
Comment (seanho96) [reply app/views/devise/shared/_links.html.erb]: Not entirely sure as well, could've been from the generator 🤔 Will remove it 👍

**Pull request merged**